test/login_test.go

```diff
@@ -99,20 +99,12 @@ func (s *CLITestSuite) TestCcloudErrors() {
 		require.Contains(tt, output, fmt.Sprintf(errors.LoggedInAsMsgWithOrg, "malformed@user.com", "abc-123", "Confluent"))
 		require.Contains(tt, output, fmt.Sprintf(errors.LoggedInUsingEnvMsg, "a-595", "default"))
 
-		output = runCommand(s.T(), testBin, []string{}, "kafka cluster list", 1)
-		require.Contains(tt, output, errors.CorruptedTokenErrorMsg)
-		require.Contains(tt, output, errors.ComposeSuggestionsMessage(errors.CorruptedTokenSuggestions))
+		_ = runCommand(s.T(), testBin, []string{}, "kafka cluster list", 1)
 	})
 
 	s.T().Run("invalid jwt", func(tt *testing.T) {
 		env := []string{fmt.Sprintf("%s=invalid@user.com", pauth.ConfluentCloudEmail), fmt.Sprintf("%s=pass1", pauth.ConfluentCloudPassword)}
-		output := runCommand(tt, testBin, env, args, 0)
-		require.Contains(tt, output, fmt.Sprintf(errors.LoggedInAsMsgWithOrg, "invalid@user.com", "abc-123", "Confluent"))
-		require.Contains(tt, output, fmt.Sprintf(errors.LoggedInUsingEnvMsg, "a-595", "default"))
-
-		output = runCommand(s.T(), testBin, []string{}, "kafka cluster list", 1)
-		require.Contains(tt, output, errors.CorruptedTokenErrorMsg)
-		require.Contains(tt, output, errors.ComposeSuggestionsMessage(errors.CorruptedTokenSuggestions))
```

Comment:
What's the reason for deleting this?

Reply:
It's not deleted but expected to see error at login. Why was invalid user able to login?

Reply:
The test before this should fail authentication and the context has changed, forbids running `login` again in next tests... This is what i observed.

Reply:
ok this is actually the new expected behavior. Logged in with a malformed token will cause error when you run other commands, and you'll be forbidden to login again as the config is corrupted and cipher will fail

Reply:
this test is not deleted, it's just expecting err=1

Reply:
If the user has an invalid JWT, they should be allowed to log in, no?

Reply:
why is that?

Reply:
ok some `if c.isTest` statements broke this test. reverted it